lms/djangoapps/discussion_api/api.py

```diff
@@ -266,6 +275,7 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
     course = _get_course_or_404(course_key, request.user)
     context = get_context(course, request)
     query_params = {
+        "user_id": unicode(request.user.id),
```

Comment:
Why is user_id added? It is not necessary for adding support for the "view" parameter, is it?

I see tests below that check that user_id is passed, but nothing that seems to be using it.